draft-ietf-mls-protocol.md

```diff
@@ -914,16 +918,16 @@ and only if the node is blank.
 
 ## Group State
 
-Each member of the group maintains a representation of the
-state of the group:
+Each member of the group maintains a partial representation of the
```

Comment:
It's not partial, it's a summary / commitment.

"Each member of the group maintains a GroupContext object that summarizes the state of the group:"